utils/jwt.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken');
+
+const createJWT = ({ payload }) => {
+	const token = jwt.sign(payload, process.env.JWT_SECRET, {
+		expiresIn: process.env.JWT_LIFETIME,
```

Comment:
JWT_LIFETIME is undefined. Either add this to both .env files or figure something else out.

Reply:
Fixed :)